pkgs/googleapis_firestore_v1/tool/update_protos.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Run this script to update to the latest google/protobuf and googleapis
+/// protobufs.
+library;
+
+import 'dart:io';
+
+void main(List<String> args) async {
+  final dartToolDir = Directory('.dart_tool');
```

Comment:
Should we consider using a more specific namespace under `dart_tool`? If we ever have a use case in `package:protobuf` to use some local file caching we might want that names available.

WDYT about `.dart_tool/googleapi_firestore/protobuf` and `.dart_tool/googleapis_firestor/googleapis`

Reply:
Sounds good! From looking around in the .dart_tool dirs, it does look like the convention is to namespace by the tool (or package) using the directory.